database/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.9'
+
+services:
+  database:
+    container_name: database
+    image: ghcr.io/bcgov/nr-spar-backend-database:test
+    environment:
+      POSTGRES_USER: ${POSTGRESQL_USER}
+      POSTGRES_PASSWORD: ${POSTGRESQL_PASSWORD}
+      POSTGRES_DB: ${POSTGRESQL_DATABASE}
+    network_mode: host
```

Comment:
You don't need to set the network to use a host configuration, just expose the ports with -p 5432:5432.